src/flinkStatementResultsManager.ts

```diff
@@ -91,6 +96,7 @@ export type PostFunction = {
  */
 export class FlinkStatementResultsManager {
   private _results: Signal<Map<string, any>>;
+  private _rawResults: Signal<StatementResultsRow[]>;
```

Comment:
These private members could use docstrings, and esp. differentiating between _results and _rawResults, (mentioning 'changlog format' in for _rawResults, and defining what the keys and values are for _results).